fastlane/Fastfile.swift

```diff
@@ -172,6 +172,7 @@ class Fastfile: LaneFile {
         registerDevice(
             name: deviceName,
             udid: deviceUDID,
+            apiKey: .userDefined(Constant.apiKey),
```

Comment:
Hmm I seem to forget iOS stuff, require API key for register device 🫣

Reply:
Yes, we need API key for register device, so we do not need to provide authentication credentials for Apple Developer Program account :D